packages/venia-concept/src/store/reducers/app.js

```diff
@@ -13,6 +14,12 @@ const reducer = (state = initialState, { payload, type }) => {
             };
         }
         default: {
+            if (error) {
+                return {
+                    ...state,
+                    error
+                };
+            }
```

Comment:
Since there are no breaks in this `switch` statement, does that mean this default case will always be reached? That is, if there is ever an `error`, this code will be executed? Interesting.